maps/tests/ApiMetadata/script.js

```diff
@@ -0,0 +1,3 @@
+WA.onInit().then(() => {
+    console.log("Metadata: ", WA.metadata);
+});
```

Comment:
The test map is useless in my opinion.

First, it should have a text associated with it (explaining we need to look at the metadata in the logs)
Second, those tests are run without an Admin API. So WA.metadata will be undefined, unless we specifically require to start with an admin.

I would rather simply modify the `tests/Metadata/getCurrentRoom.json` to add a console log of the metadata with a small comment explaining that metadata are filled only if an admin is started and provides some.